inc/saiswitch.h

```diff
@@ -1213,10 +1311,8 @@ typedef sai_status_t (*sai_get_switch_attribute_fn)(
  */
 typedef struct _sai_switch_api_t
```

Comment:
great, example with be helpful. 
@AshokDaparthi are you planning to provide some sample code that uses new API's?